### PR TITLE
#5117 Change 0.10.0-SNAPSHOT to 0.10.0 for 0.10.x branch 

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5117 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
 group=oharastream.ohara
 # jfrog doesn't accept -SNAPSHOT. Please use rc0 instead if you want to test publish
-version=0.10.0-SNAPSHOT
+version=0.10.0
 # following properties are used by publish task
 userOrg=oharastream
 repoName=ohara
 # git branch name or tag name, eg. master, 0.9.x
-branch=0.10.x
+branch=0.10.0
```
Comment (vitojeng) [inline gradle.properties:24]: branch value should be `0.10.x`

Comment (eechih) [reply gradle.properties:24]: After discussing with @vitojeng @konekoya, the value is still 0.10.0

**Comment by eechih** [comment]
@oharastream/developer  Please help review, thanks.

**Comment by eechih** [comment]
> Could you help to replace the old version string 0.9.0 in our project also?

No problem

**Pull request merged**